spec/decorators/video_decorator_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe VideoDecorator do
+
+  it 'responds to methods which a Video instance responds to' do
+    decorator = Video.new.decorate
+    expect(decorator).to respond_to(:category)
+    expect(decorator).to respond_to(:reviews)
+  end
+
+  it 'responds to method :total_rating' do
```

Comment:
You are testing the implementation of this method below, so you will know if the decorator doesn't respond to the method when you use it there. You probably don't need this spec.